add foto de un gato

Z7NR opened the pull request:
Añadí la foto de un gato triste con una paupérrima calidad.

Z7NR committed:
add foto de un gato

Añadí la foto de un gato triste con una paupérrima calidad.

DelvisSanabria reviewed (changes requested):
Cambia los nombres correctamente

Z7NR committed:
mod nombres de archivos

Modifique los nombres de la carpeta y de la imagen

Z7NR committed:
Mod

Modifique el nombre del archivo, borre la foto del gato debido a su extension y coloque otra foto de un gato con la extension correcta
